### ISSUE TITLE
Conky doesn't fallback to another font.

### ISSUE THREAD

**Issue by bad-interpreter** [brndnmtthws/conky#484]
If I set `font = 'Droid Sans:size=9'` in options, it doesn't fallback to "Droid Sans Fallback" font if the glyph in unavailable. 
I guess that's actually not a bug, but I'd love to see this implemented.

**Comment by godalming123** [comment]
I agree I would like to add icons into my conky so I use a matirial icon font only to discover its normal letters look ugly as making it unusable.

**Comment by godalming123** [comment]
Issue not stale!

**Comment by Caellian** [comment]
> I would like to add icons into my conky so I use a matirial icon font only to discover its normal letters look ugly as making it unusable

You can specify multiple fonts with `font`, `font0`, `font1`, ..., `font9` and switch between them with commands (Default font `${font5}` fifth font) in `conky.text` for using a different font (e.g. for icons).

While font fallback support _would_ fix your issue, just noting there's a way of dealing with your use case already.

<hr/>

Font fallback with xft [example](https://stackoverflow.com/questions/77989704/how-to-use-fallback-font-to-draw-text-with-xlib-libxft).

It would be nice to allow `font<n>` options to take either a string or a list of strings for fallbacks, load all of them, and then try each until one containing the glyph is found.

**Comment by k0zyrev** [comment]
still hoping for the fallback implementation!

**Comment by AnapalOne** [comment]
bumping potential fallback feature! 

I use conky in a way that the text switches dynamically between different writing systems (i.e. CJK, Arabic), sometimes multiple in one line, and this would be a convenient feature to have. hoping for any updates regarding this!
